cmake/modules/RootBuildOptions.cmake

```diff
@@ -131,6 +131,7 @@ ROOT_BUILD_OPTION(fcgi OFF "Enable FastCGI suppport in HTTP server")
 ROOT_BUILD_OPTION(imt ON "Enable support for implicit multi-threading via Intel® Thread Bulding Blocks (TBB)")
 ROOT_BUILD_OPTION(jemalloc OFF "Use jemalloc memory allocator")
 ROOT_BUILD_OPTION(libcxx OFF "Build using libc++")
+ROOT_BUILD_OPTION(liburing OFF "Enable support for io_uring (requires Linux kernel >= 5.1)")
```

Comment:
Maybe the name ```uring``` for option is better? (consistent with all other, e.g. such as jemalloc)

Reply:
Thank you, done in 10999c0